pkg/controller/controlplane/valuesprovider.go

```diff
@@ -455,7 +455,7 @@ func (vp *valuesProvider) getConfigChartValues(
 		"serverName":   serverName,
 		"serverPort":   port,
 		"insecureFlag": region.VsphereInsecureSSL,
-		"datacenters":  strings.Join(helper.CollectDatacenters(region), ","),
+		"datacenters":  helper.CollectDatacenters(region),
```

Comment:
Why are there helper packages called `helper` and `helpers` i the same project.

Reply:
Nice hint. Will note it for a refactoring task, but not in this PR.